utils/docker/vcluster/daos-cm.sh

```diff
@@ -231,7 +234,7 @@ done
 CMD="$1"
 DAOS_IFACE_IP="$2"
 
-cd "$CWD"
+cd "$CWD" || exit 1
```

Comment:
`set -e` would be much more encompassing and thorough.

Reply:
I agree but without being able to run/test this code I didn't want to do that.  @knard-intel any thoughts here?

Reply:
After reviewing the code, I do not remember why I do not have put `-e` with the `-o pipefail`.
Seems OK for me to put the `-e` for me.
I am working on a PR where I will manually test this script.
In conclusion, using the `-e` seems OK for me.

Reply:
I had to suppress a couple of things in this directory I'd prefer not to have done, if you are able to take a closer look when doing future work in this area that would be great.